### ISSUE TITLE
[Idea] Reduce Form Recognizer location deployment related errors

### ISSUE THREAD

**Issue by pamelafox** [Azure-Samples/azure-search-openai-demo#574]
Form Recognizer is only available in ~70% of regions, see https://azure.microsoft.com/en-us/explore/global-infrastructure/products-by-region/?products=cognitive-services and search for "Document Intelligence".

I'm considering a few options for increasing the deployability:

1) Restrict the overall location based on locations where form recognizer is available

2) Add a location specifically for just form recognizer with just its regions (that increases the number of things to fill out though)

**Comment by chuwik** [comment]
Good point! Option 1 sounds good to me, feels better than having to fill out more option.

**Comment by pamelafox** [comment]
Option 1 doesn't currently work, I've asked azd team, so we may need to go for option 2 if that's not possible.

**Comment by pamelafox** [comment]
Filed https://github.com/Azure/azure-dev/issues/2695

**Comment by pamelafox** [comment]
We have since added a location variable just for Document Intelligence, so this issue can be closed.
